spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
+require 'emerald/parser'
+require 'emerald/compiler'
+
```

Comment:
I think my testing style has become a little inconsistent - I was trying to do more BDD style tests here so added context. Having really been doing that so far, might update the other tests so that the styles are aligned at a later stage